bundles/wps/wps-service/src/test/java/org/orbisgis/wpsservice/controller/execution/DataProcessingTest.java

```diff
@@ -0,0 +1,262 @@
+        //Check if the resulting object is a string containing the representation of the bounding box
+        Object result2 = dataMap.get(uri);
+        Assert.assertTrue("The object resulting of the postprocessing of the bounding box should be a String.",
+                result2 instanceof String);
+        String str = result2.toString();
+        Assert.assertEquals("The bounding box geometry wasn't the one expected.",
+                ":4326;0,0,1,1", str);
+    }
+
+    /**
+     * Test the preprocessing of a 3D bounding box (now not supported). It should return a null value.
+     * @throws MalformedScriptException
+     */
+    @Test
+    public void test3DBoundingBoxProcessing() throws MalformedScriptException {
+        //Store into the dataMap the boundingBox data
+        URI uri = URI.create(UUID.randomUUID().toString());
+        Map<URI, Object> dataMap = new HashMap<>();
+        dataMap.put(uri, "EPSG:4326;0,0,0,1,1,1");
+        //Generate the InputDescriptionType object containing the boundingBox
+        List<Format> formatList = FormatFactory.getFormatsFromExtensions(new String[]{".txt"});
+        BoundingBoxData boundingBoxData = new BoundingBoxData(formatList, "EPSG:4326",
+                new String[]{"EPSG:4326","EPSG:2000", "EPSG:2001"}, 2);
+        CodeType identifier = new CodeType();
+        identifier.setValue(uri.toString());
+        InputDescriptionType input = new InputDescriptionType();
+        input.setIdentifier(identifier);
+        input.setDataDescription(objectFactory.createBoundingBoxData(boundingBoxData));
+        //Preprocess the bounding box
+        dataProcessingManager.preProcessData(input, dataMap, null);
+        //Check if the resulting object is null
+        Object result = dataMap.get(uri);
+        Assert.assertTrue("The object resulting of the preprocessing of the bounding box should be null.",
+                result == null);
+    }
+
+    /**
+     * Test the preprocessing of an incoming GeometryData and if the geometry generated is correct.
+     * Then check its postprocessing and if its string representation is correct.
+     * @throws MalformedScriptException
+     */
+    @Test
+    public void testGeometryProcessing() throws MalformedScriptException {
+        //Store into the dataMap the geometryData data
+        URI uri = URI.create(UUID.randomUUID().toString());
+        Map<URI, Object> dataMap = new HashMap<>();
+        dataMap.put(uri, "POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))");
+        //Generate the InputDescriptionType object containing the GeometryData
+        List<Format> formatList = FormatFactory.getFormatsFromExtensions(new String[]{".txt"});
+        List<DataType> geometryList = new ArrayList<>();
+        geometryList.add(DataType.POLYGON);
+        GeometryData geometryData = new GeometryData(formatList, geometryList);
+        geometryData.setExcludedTypeList(new ArrayList<DataType>());
```

Comment:
so it is not necessary to add geometryData.setExcludedTypeList(new ArrayList<DataType>());